crates/common/src/term/logger.rs

```diff
@@ -0,0 +1,117 @@
+use std::fmt::Display;
```

Comment:
@ftheirs is this file used at all?

Reply:
Yeap! This one is used! If you check `Cmd` there are multiple calls. Probably, we are not using yet all the available functions but it's important